Products/CMFPlone/browser/login/templates/mail_password_template.pt

```diff
@@ -33,7 +33,7 @@ The following link will take you to a page where you can reset your password for
      i18n:translate="mailtemplate_tracking_information"
      tal:condition="isAnon">
 If you didn't expect to receive this email, please ignore it. Your password has not been changed.
-Request made from IP address <tal:i18n tal:define="host request/HTTP_X_FORWARDED_FOR|request/REMOTE_ADDR"
+Request made from IP address <tal:i18n tal:define="host python:request.HTTP_X_FORWARDED_FOR or request.REMOTE_ADDR"
```

Comment:
I tried locally `request.HTTP_X_FORWARDED_FOR` or even `request.ATTR_THAT_DONT_EXIST`, both are false and this fallbacks to `request.REMOTE_ADDR` like it is supposed to.